elements/pl-workspace/pl-workspace.py

```diff
@@ -0,0 +1,13 @@
+import chevron
+
+
+def render(element_html, data):
+    # Get workspace url
+    workspace_url = data['options']['workspace_url']
```

Comment:
Maybe not necessary for the MVP, but it'd be nice to do something different if `workspace_url` isn't defined (i.e., the question doesn't have a workspace set up). Either throwing an error, or rendering a red box/button/something to alert course staff to the fact that something is wrong. In general, failing loudly/brightly is better to help people catch their mistakes sooner.

Reply:
Added `TODO` comment

Reply:
nit, including the full URL is preferable since that allows you to quickly cmd-click from your IDE to get to the comment.